src/Tags/Structure.php

```diff
@@ -75,6 +78,46 @@ protected function ensureStructureExists(string $handle): void
         throw_unless(Nav::findByHandle($handle), new NavigationNotFoundException($handle));
     }
 
+    protected function query($handle)
+    {
+        $query = new ItemQueryBuilder();
+
+        return Str::startsWith($handle, 'collection::')
+            ? $this->queryForCollection($query)
+            : $this->queryForNav($query);
+    }
+
+    protected function queryForCollection($query)
+    {
+        if (! $this->isQueryingStatus()) {
+            $this->queryStatus($query);
+        }
+        $this->queryConditions($query);
+
+        return $query;
+    }
+
+    protected function queryForNav($query)
+    {
+        return $this->queryStatus($query);
```

Comment:
If we're not going to allow filtering on navs, why don't we get rid of this too?

Reply:
We still want unpublished entries filtered out by default.